lib/pbench/agent/tool_meister_start.py

```diff
@@ -801,15 +800,43 @@ def start(_prog: str, cli_params: Namespace) -> int:
             ", ".join(_orchestrate_choices),
         )
         return ReturnCode.INVALIDORCHESTRATE
+    if cli_params.orchestrate == "create":
+        orchestrate = True
+    else:
+        assert (
+            cli_params.orchestrate == "existing"
+        ), f"Unexpected --orchestrate parameter, {cli_params.orchestrate!r}"
+        if cli_params.redis_server is None or cli_params.tool_data_sink is None:
```

Comment:
Perhaps
```
        if not cli_params.redis_server or not cli_params.tool_data_sink:
```
because any kind of false-y value there would be bad.

Reply:
Just moved existing code.  PRs to follow up with a change welcome!